packages/makeup-menu-button/src/index.js

```diff
@@ -117,8 +121,14 @@ function _onMenuKeyDown(e) {
 }
 
 function _onMenuItemSelect(e) {
-  if (this._buttonPrefix && e.detail.el.getAttribute("role") === "menuitemradio") {
-    this._buttonTextEl.innerText = `${this._buttonPrefix} ${e.detail.el.innerText}`;
+  if (e.detail.el.getAttribute("role") === "menuitemradio") {
```

Comment:
Sorry, the other comments were on the files in the `dist` folder, but the same would apply here.

Reply:
Unfortunately, we cannot return early as weather the condition is met or not, we still need to execute L134 here.

Reply:
`const` should typically be defined at the top of functions, so just move that up to the top and you should be able to do a return.